src/encoded/audit/experiment.py

```diff
@@ -2758,32 +2758,16 @@ def audit_experiment_biosample_term(value, system, excluded_types):
                 continue
 
             biosample = lib['biosample']
-            bs_type = biosample.get('biosample_ontology', {}).get('classification')
-            bs_name = biosample.get('biosample_ontology', {}).get('term_name')
-            bs_id = biosample.get('biosample_ontology', {}).get('term_id')
-
-            if bs_type != term_type:
+            bs_type = biosample.get('biosample_ontology', {}).get('@id')
+            bs_name = biosample.get('biosample_ontology', {}).get('name')
+            experiment_bs_type = value.get('biosample_ontology', {}).get('@id')
+            experiment_bs_name = value.get('biosample_ontology', {}).get('name')
+            if bs_type != experiment_bs_type:
                 detail = 'Experiment {} '.format(value['@id']) + \
                          'contains a library {} '.format(lib['@id']) + \
-                         'prepared from biosample type \"{}\", '.format(bs_type) + \
+                         'linked to biosample type \"{}\", '.format(bs_name) + \
```

Comment:
Since we're changing this line let's clean up the string formating.  No slashes and one format.  Use double quotes for the outer most and single quotes for the internal ones.

Reply:
Sounds good, I'll edit and update the PR